dependency_support/embedded_python_interpreter/print_libpython_path.py

```diff
@@ -23,4 +23,13 @@
     for pv in ('LIBDIR', 'LIBPL')
 ]
 
+if 'rules_hdl_cpython' in config_vars['prefix']:
```

Comment:
is there a reference bug for this workaround?

Reply:
It looks like we can remove this workaround.